argocd-add-status-badge/action.yaml

```diff
@@ -0,0 +1,61 @@
+name: Attach wait for healthy
+description: Attaches the current PR to an asana task, if a task can be identified
+
+inputs:
+  github-token:
+    description: A github token for the action
+    required: true
+  argo-host:
+    description: ArgoCd host
+    required: false
+    default: https://argocd.dev.clusters.neohelden.com
+  argo-app:
+    description: ArgoCD app being used as base for deploying the built container image
+    required: false
+    default: neap
+  workspaces-base-domain:
+    description: Domain name of the workspaces base
+    required: false
+    default: "dev.clusters.neohelden.com"
+  argo-suffix:
+    description: Suffix for the ArgoCD app name
+    required: false
+    default: ""
+  app-prefix:
+    description: Prefix for the ArgoCD App URL name
+    required: false
+    default: "server"
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Comment on PR
+      uses: actions/github-script@v5
+      env:
+        PR_ADDITION: |
+          :tada: This PR is an App Deployment :tada: 
+          [![App Status](${{ inputs.argo-host }}/api/badge?name=${{ inputs.argo-app }}&revision=true)](${{ inputs.argo-host }}/applications/${{ inputs.argo-app }})
+          View the app at: https://${{ inputs.app-prefix }}${{ inputs.argocd-suffix }}.${{ inputs.workspaces-base-domain }}
+          The badge was added automatically. 
+
+          ---
+          Please do not edit above this line!
```

Comment:
```suggestion
          The badge was added automatically. 

          _Please edit below this line only!_
          ---
          
```

Reply:
Really? The Action currently listens for `Please do not edit above this line!`